paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/resources/RepositoryLoader.kt

```diff
@@ -284,22 +295,22 @@ abstract class RepositoryLoader<T : LoadableResourceRepository>(
     configuration: RepositoryConfiguration,
     shouldParseResourceIds: Boolean
   ) {
-    val folderType = getFolderType(file)
-    if (folderType == VALUES) {
+    if (folderInfo.resourceType == null) {
       if (isXmlFile(file)) {
         parseValueResourceFile(file, configuration)
       }
     } else {
-      if (isXmlFile(file) && FolderTypeRelationship.isIdGeneratingFolderType(folderType)) {
+      if (shouldParseResourceIds && folderInfo.isIdGenerating && isXmlFile(file)) {
         parseIdGeneratingResourceFile(file, configuration)
       }
 
-      parseFileResourceFile(file, folderType, configuration)
+      val item = createFileResourceItem(file, folderInfo.resourceType, configuration)
+      addResourceItem(item)
     }
   }
 
   @Suppress("UNCHECKED_CAST")
-  private fun addResourceItem(item: BasicResourceItem) {
+  private fun addResourceItem(item: BasicResourceItemBase) {
```

Comment:
```suggestion
  private fun addResourceItem(item: BasicResourceItem) {
```

We still want to merge BasicResourceItem with BasicResourceItemBase

Reply:
addressed in https://github.com/cashapp/paparazzi/pull/893